runtime/ftplugin/zig.vim

```diff
@@ -50,19 +50,10 @@ if !exists('g:zig_std_dir') && exists('*json_decode') &&
 endif
 
 if exists('g:zig_std_dir')
```

Comment:
`g:zig_std_dir` should also be documented

Reply:
Fixed